io.github.mihnearadulescu.imagefanreloaded.json

```
@@ -0,0 +1,45 @@
+{
+    "app-id" : "io.github.mihnearadulescu.imagefanreloaded",
```

```suggestion
    "app-id" : "io.github.mihnea_radulescu.imagefanreloaded",
```
This is what the app-id should be.

---

Updated, thanks!